src/seedsigner/controller.py

```diff
@@ -123,6 +128,10 @@ def configure_instance(cls, disable_hardware=False):
         # TODO: Rename "storage" to something more indicative of its temp, in-memory state
         controller.storage = SeedStorage()
         controller.settings = Settings.get_instance()
+        
+        controller.microsd = MicroSD.get_instance()
+        controller.microsd.settings_handler = controller.settings.microsd_handler
```

Comment:
Same rationale as the `ui_handler`; `MicroSD` may as well just have direct knowledge of how to invoke its one-and-only `settings_handler`. `Settings` is a Singleton, so `MicroSD` can `get_instance()` the settings itself.